03.asynchronous/callback.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+import sqlite3 from "sqlite3";
+const db = new sqlite3.Database(":memory:");
+
+db.run(
+  "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)",
+  function () {
+    db.run("INSERT INTO books(title) VALUES('cherry')", function () {
```

Comment:
CREATE TABLE と SQL のスタイルを揃えるなら以下のようになると思います。

```suggestion
    db.run("INSERT INTO books (title) VALUES ('cherry')", function () {
```

Reply:
ご指摘ありがとうございます！
なるほどです！
細かい視点を教えてくださりありがとうございます✨